distributions/bedrock/run.yaml

```diff
@@ -35,12 +35,12 @@ providers:
         region_name: <AWS_REGION>
   agents:
     - provider_id: meta0
-      provider_type: meta-reference
+      provider_type: inline::meta-reference
```

Comment:
Is this inline::faiss?

Reply:
@raghotham this is the agents API so certainly not faiss. it is our _agentic loop_

Reply:
Oops